test-network-function/networking/suite.go

```diff
@@ -167,16 +167,19 @@ func processContainerIpsPerNet(containerID *configsections.ContainerIdentifier,
 
 // runNetworkingTests takes a map netTestContext, e.g. one context per network attachment
 // and runs pings test with it
-func runNetworkingTests(netsUnderTest map[string]netTestContext, count int) {
+func runNetworkingTests(netsUnderTest map[string]netTestContext, count int) (badNets map[string][]string) {
```

Comment:
Instead of declaring `badNets` in the function's return value, on 177 just use `badNets := make(map[string][]string)`.

Reply:
Done, by I used badNets := map[string][]string{} instead.